js-miniapp-bridge/src/types/token-data.ts

```diff
@@ -2,15 +2,35 @@
 export interface NativeTokenData {
   token: string;
   validUntil: number;
+  scopes?: NativeTokenPermission;
+}
+
+/** @internal */
+export interface NativeTokenPermission {
+  audience?: string;
+  scopes?: string[];
 }
 
 /** Token data type. */
 export class AccessTokenData {
   readonly token: string;
   readonly validUntil: Date;
+  readonly scopes: AccessTokenPermission;
```

Comment:
See my above comment about the return format for audience/scopes. I think it will be easier to use if they can access these properties directly under `AccessTokenData`.

Reply:
I think I did the necessary